actionview/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Prevent non-primary mouse keys from triggering Rails UJS click handlers.
+    Firefox fires click events even if the click was triggered by non-primary mouse keys such as right- or scroll-wheel-clicks.
+    For example, right-clicking a link such as the one described below (with an underlying ajax request registered on click) should not cause that request to occur.
+
+        <%= link_to 'Remote', remote_path, class: 'remote', remote: true, data: { type: :json } %>
```

Comment:
Nitpick: Code examples in the changelog should have backticks for markdown code blocks.